tests/commands/test_changelog_command/test_breaking_change_content_v1_with_exclamation_mark.md

```diff
@@ -0,0 +1,5 @@
+## Unreleased
+
+
+- drop support for py36
+
```

Comment:
What do you think of this @Lee-W @jenstroeger ?

It won't show the title `### BREAKING CHANGE`. But it's the best I could do with the new regex, I cannot repeat the `breaking` group in here:

https://github.com/commitizen-tools/commitizen/pull/728/files#diff-c951de16d627d82e01eb3ce8d7c4f0b28973ffe147520620762e7140ede2dd80R115